Backpack/Calendar/Classes/BPKCalendar.h

```diff
@@ -16,6 +16,8 @@
  * limitations under the License.
  */
 
+#import "BPKSimpleDate.h"
```

Comment:
`BPKSimpleDate` can be forward declared in this file I believe

Reply:
I was going with that first, but then whoever uses the calendar needs to import one more thing on their own, is that preferred?

Reply:
People who use the calendar should be importing `Backpack/Calendar.h` not `Backpack/BPKCalendar.h`. We should add `BPKSimpleDate.h` to `Calendar.h`